test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/Pagination/RangeValidationTests.cs

```diff
@@ -0,0 +1,152 @@
+using System.Net;
+using System.Threading.Tasks;
+using Bogus;
+using FluentAssertions;
+using JsonApiDotNetCore.Configuration;
+using JsonApiDotNetCore.Serialization.Objects;
+using JsonApiDotNetCoreMongoDbExample;
+using JsonApiDotNetCoreMongoDbExample.Models;
+using Microsoft.Extensions.DependencyInjection;
+using MongoDB.Driver;
+using Xunit;
+
+namespace JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.Pagination
+{
+    public sealed class RangeValidationTests : IClassFixture<IntegrationTestContext<Startup>>
```

Comment:
All tests, _except_ the following, can be removed, because they test query string validation logic.
- When_page_number_is_too_high_it_must_return_empty_set_of_resources
- When_page_size_is_zero_it_must_succeed
- When_page_size_is_positive_it_must_succeed